File: packages/insomnia-httpsnippet/test/fixtures/output/node/native/https.js
var http = require("https");

var options = {
  "method": "GET",
  "hostname": "mockbin.com",
  "port": null,
  "path": "/har",
  "headers": {}
};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();

File: packages/insomnia-httpsnippet/test/fixtures/output/go/native/https.go
package main

import (
	"fmt"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "https://mockbin.com/har"

	req, _ := http.NewRequest("GET", url, nil)

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}
